include/fast_float/float_common.h

```diff
@@ -63,16 +67,24 @@ template <typename UC> struct from_chars_result_t {
 using from_chars_result = from_chars_result_t<char>;
 
 template <typename UC> struct parse_options_t {
-  constexpr explicit parse_options_t(chars_format fmt = chars_format::general,
-                                     UC dot = UC('.'), int b = 10)
-      : format(fmt), decimal_point(dot), base(b) {}
+  FASTFLOAT_CONSTEXPR20 explicit parse_options_t(chars_format fmt = chars_format::general,
+                                     UC dot = UC('.'), const int b = 10) noexcept
+      : format(fmt), decimal_point(dot), base(static_cast<uint8_t>(b)) {}
 
   /** Which number formats are accepted */
-  chars_format format;
+  const chars_format format
+      // adjust for deprecated feature macros
+#ifdef FASTFLOAT_ALLOWS_LEADING_PLUS
+                            | chars_format::allow_leading_plus
```

Comment:
This won't compile, and even if it did, it will be overridden by `format(fmt)` in the constructor list.

Reply:
Fix for this https://github.com/fastfloat/fast_float/pull/307/commits/f496321570c6ad7e12dd309934217b6eb3c30805